# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vdr/dxr3-0.2.2-ffmpeg.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- dxr3-0.2.2/Makefile.orig	2004-01-04 18:32:10.000000000 +0100
+++ dxr3-0.2.2/Makefile	2005-04-07 13:28:49.000000000 +0200
@@ -42,8 +42,8 @@
 
 ### Includes and Defines (add further entries here):
 
-INCLUDES += -I$(VDRDIR)/include -I$(DVBDIR)/include -I$(FFMDIR)
-LIBS     = -L$(FFMDIR)/libavcodec -lavcodec
+INCLUDES += -I$(VDRDIR)/include -I$(DVBDIR)/include -I/usr/include/ffmpeg
+LIBS     = -lavcodec
 DEFINES += -DPLUGIN_NAME_I18N='"$(PLUGIN)"'
 DEFINES += -DSOCKET_CHMOD=0660
 DEFINES += -D_GNU_SOURCE
--- dxr3-0.2.2/dxr3audiodecoder.h.orig	2005-04-07 13:51:42.000000000 +0200
+++ dxr3-0.2.2/dxr3audiodecoder.h	2005-04-07 13:51:58.000000000 +0200
@@ -10,7 +10,7 @@
 #define __DXR3AUDIODECODER_H
 
 extern "C" {
-#include "libavcodec/avcodec.h"
+#include "ffmpeg/avcodec.h"
 };
 
 #include <stdlib.h>
